src/release/index.md

```diff
@@ -39,7 +39,7 @@ The following table provides the dates for each scheduled release in 2020 and 20
 | Quarter             | Versions                   | General Availability | Pre-release        | Beta               |
 |---------------------|----------------------------|----------------------|--------------------|--------------------|
 | 2020 Q3             | 2.4.0<br>2.3.5-p2          | July 28, 2020        | None               | June 8, 2020       |
-| 2020 Q4             | 2.4.1<br>2.3.6             | October 15, 2020     | October 15, 2021   | September 10, 2020 |
+| 2020 Q4             | 2.4.1<br>2.3.6             | October 15, 2020     | October 15, 2020   | September 10, 2020 |
```

Comment:
Hmm, is this correct? Shouldn't it be 14 days ahead of 15 October, so 1 October?

Reply:
Sorry, looks like I'm having a day @hostep... Yes, you're right.